### PR TITLE
IA-1339 persist WORKSPACE_BUCKET

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-1339

Tested with Azmina's terra-ui branch

Screen Shot 2019-09-11 at 7 43 55 PM

### CODE REVIEW

File: server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/StorageLinksService.scala [hunk 1]
```diff
@@ -64,23 +77,32 @@ class StorageLinksService(storageLinks: StorageLinksCache, googleStorageAlg: Goo
     } yield link
   }
 
+  private def persistWorkspaceBucket(baseDirectory: LocalDirectory, cloudStorageDirectory: CloudStorageDirectory): IO[Unit] = {
+    val fileBody =
+      RuntimeVariables(s"gs://${cloudStorageDirectory.bucketName.value}/notebooks") //appending notebooks to mimick old jupyter image until we start using new images.
```
Comment (rtitle) [inline server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/StorageLinksService.scala]: This is to make it work with the current logic in `kernel-bootstrap.sh`?

Comment (Qi77Qi) [reply server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/StorageLinksService.scala]: yes

Comment (rtitle) [reply server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/StorageLinksService.scala]: Also I thought the old format of `.delocalize.json` was something like:
```
{
  "destination": "gs://foo/notebooks"
}
```
Does this output the same format?

Comment (rtitle) [reply server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/StorageLinksService.scala]: Ah nvm I see `runtimeVariablesEncoder` -- looks good.

**Pull request merged**